src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -490,5 +506,38 @@ private HashSet<string> GetRequestedExtensions(string exts, bool fallback)
 
             return [];
         }
+
+        private string GetCacheBustingMediaPublicUrl(string path) =>
```

Comment:
=> .... on a new line

Reply:
Why is that better?

Reply:
just for consistence with what we do everywhere else.

Reply:
Hmm, this is at least inconsistent, then. Because there are a lot of same-line cases:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/2b4e9ff9-ea8a-4b2a-b043-2422c3c57e23)

Some of these are lambdas or one-liners, but there are a lot of multi-line methods too.

I'm not trying to get stuck on this, but there does seem to be disagreement in the codebase. https://github.com/OrchardCMS/OrchardCore/issues/7950 can fix this BTW.

Reply:
Yes. Just an initiate that I have been doing to keep consistency in every file I edit. If you don't agree, feel free to resolve the comment and move on. It's a personal preference at the end.

Reply:
I think let's wait with this until we have a rule agreed on and consistently applied, as well as checked by analyzers.